scripts/deduplicate_index_urls.py

```diff
@@ -277,11 +292,28 @@ def main():
         format='%(asctime)s - %(process)s - %(levelname)s - %(message)s'
     )
 
+    if args.export_urls_file is None:
+        url_list_output_file = os.devnull
```

Comment:
This is already handled in the function itself.

Reply:
Yes, I know.

If I remove this from here, the very next line will need to check for None anyways, in order not to run into an error by trying None.exists().

So I will remove it from the filter_file() function. Just keep in mind that if in the future we want to call that function from elsewhere, we have to pay attention to this None problem.